packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -65,6 +71,20 @@ type Props = {
      * Change the field’s sub-components to fit a dark background.
      */
     light?: boolean;
+    /**
+     * The object containing the custom labels used inside this component.
+     *
+     * This is useful for internationalization. Defaults to English.
+     */
+    labels?: LabeledFieldLabels;
+};
+
+type LabeledFieldLabels = {
+    errorIconAriaLabel: string;
+};
```

Comment:
**suggestion:** This probably needs to be exported so consumers can use the type if they need it (it's probably fine as it is b/c the component instance should be able to infer the types, but just making sure that this case will be covered).